mindmeld/bot/webex_bot_server.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module contains the Webex Bot Server component.
+"""
+
+import logging
+import json
+from flask import request
+import requests
+from ciscosparkapi import CiscoSparkAPI
+
+CISCO_API_URL = 'https://api.ciscospark.com/v1'
+ACCESS_TOKEN_WITH_BEARER = 'Bearer '
+
+
+class WebexBotServer:
+    """
+    A sample server class for Webex Teams integration with any MindMeld application
+    """
+
+    def __init__(self, app, webhook_id, access_token, conv):
+        self.app = app
+        self.webhook_id = webhook_id
+        self.access_token = access_token
+        self.conv = conv
+
+        self.logger = logging.getLogger(__name__)
+
+        if not self.webhook_id:
+            raise Exception('WEBHOOK_ID not set')
+        if not self.access_token:
+            raise Exception('BOT_ACCESS_TOKEN not set')
+
+        self.spark_api = CiscoSparkAPI(self.access_token)
+        self.access_token_with_bearer = ACCESS_TOKEN_WITH_BEARER + self.access_token
+
+        @self.app.route('/', methods=['POST'])
+        def handle_message():
+            me = self.spark_api.people.me()
+            data = request.get_json()
+
+            for key in ['personId', 'id', 'roomId']:
+                if key not in data['data'].keys():
+                    return 'BAD REQUEST', 400, {'message': 'personId/id/roomID key not found'}
```

Comment:
Convert such messages into the following:
`BAD_REQUEST_NAME = 'BAD REQUEST'`
`BAD_REQUEST_CODE = 400`

and then utilize them.